README.md

```diff
@@ -6,15 +6,33 @@ That's what it is.
 /dt [code <your code here>]
 ```
 
+# Operator Notes
+
+[Factor 0.99](https://re.factorcode.org/2023/08/factor-0-99-now-available.html) and Docker (or Podman) are required.
+
+Run:
+
+```shell
+APPLICATION_ID=xxxxxx TOKEN=yyyyyy /where/ever/is/factor -roots=$PWD runbot.factor
+```
+
+Run, Val's cursed dev environment edition:
+
+```shell
+dotenv run flatpak run --env=DOCKER_CMD=podman org.factorcode.Factor -roots=$PWD runbot.factor
+```
+
+Invite to a guild:
+
+in the "OAuth2 URL Generator" dev console section, select **only** the `bot` scope, no need for anything else.
+Go to the generated URL, select a guild, confirm.
+
 # Links
 
 * https://discord.com/developers/docs/getting-started
 * https://hub.docker.com/r/booniepepper/dt
-* https://github.com/shardlab/discordrb
+* https://docs.factorcode.org/content/vocab-discord.html
 
 # Etc
 
-Why Ruby? It was the simplest _and_ easiest discord bot library that supported
-slash actions. If I didn't need that, the Lua thing looked pretty nice.
-
-A 2023 side quest of [J.R. Hill](https://so.dang.cool)
+Commissioned by [J.R. Hill](https://so.dang.cool/), made by [Val Packett](https://val.packett.cool/).
```

Comment:
I'll likely add small tweaks and changes over time and update the wording, but I'll keep your link here.